chapters/packages.tex

```diff
@@ -321,3 +321,135 @@ \section{External Resources}\label{external-resources}
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-03-15 10:52+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+\end{lstlisting}
+All general terms in the header should be replaced by specific information.
+
+Following the header, there is one translation entry for each string to be translated.
+It can start with an optional comment describing the location (file name and line number) of the text to translate.
+Multiple occurences of the same string can be listed here, separated by space.
+\end{nonnormative}
+
+The $\mathit{messageContext}$ following the keyword \lstinline!msgctxt! shall be the full name of the Modelica class (e.g., \lstinline!"Modelica.Blocks.Math.Sin"!) where the text appears.
+Short class definitions do not appear here.
+Texts in such classes belong to the enclosing full class definition.
+
+The text string which shall be translated is used as $\mathit{messageIdentifier}$ (following the \lstinline!msgid! keyword), and shall contain the original string from the Modelica code.
+% Describe down-quoting and concatenation of Modelica string literals.
+Note that if a \lstinline!msgid! string is given more than once in the same context, all occurrences are translated with the same (last) translation!
+
+\begin{nonnormative}
+The $\mathit{messageTranslation}$ (following the keyword \lstinline!msgstr!) is the translation of $\mathit{messageIdentifier}$ and is typically edited using special tools for translators.
+In the template file this string is empty by definition.
+If this is empty in a language specific file the $\mathit{messageIdentifier}$ may be used instead.
+\end{nonnormative}
+
+\begin{nonnormative}
+Since in Modelica control sequences also start with a backslash and another backslash is used to use sequences literally or to hide double quotes, no change is required here.
+But Modelica allows strings to go over more than one line, gettext does not.
+Therefore, line breaks in Modelica must be encoded with \lstinline!"\n"! for gettext.
+
+In order to avoid long lines in the translated strings (following \lstinline!msgid! or \lstinline!msgstr!), strings may be split into smaller parts given on consecutive lines.
+E.g., the Modelica description
+\begin{lstlisting}
+"A
+B\"C" + "D\nE"
+\end{lstlisting}
+evaluates to:\par
+\hspace{1em}\begin{minipage}{0.5\textwidth}
+A\\
+B"CD\\
+E
+\end{minipage}\par
+which in the translation entry looks like:
+\begin{lstlisting}
+msgid: ""
+"A\n"
+"B\"CD\n"
+"E"
+\end{lstlisting}
+\end{nonnormative}
+
+\begin{nonnormative}
+To support the translation of these strings a number of free and commercial tools exist in context of GNU gettext.
+A Modelica tool should provide a function to create the initial template file from an existing Modelica library.
```

Comment:
```suggestion
```
Just remove this, as it doesn't seem important to specify. Clearly it is a good idea, but it can also be done externally (so "loaded" doesn't make sense) - or perhaps implemented in some other way.